src/typescript/06-types/types.md

```diff
@@ -248,7 +248,7 @@ function getTotalDistanceAbleToTravelInWater(
 }
 ```
 
-This works, but it’d be a pain to have to figure out which dinosaur we have and then select the correct function to use for that dinosaur group. It’d be nice if we could just pass our dinosaurs to a single function and get the result we’re after.
+This works, but it’d be a pain to have to figure out which dinosaur we have and then select the correct function to use for that dinosaur group. It’d be nice if we could pass our dinosaurs to a single function and get the result we’re after.
 
 Currently, our typing is the limitation stopping us from achieving this. This is where type unions can make a big impact in our code. Type unions use the `|`, operator to combine any number of different types and can be read as **or**. This allows us to create a Dinosaur type that is either a `LandDinosaur`, an `AirDinosaur`, or a `WaterDinosaur`.
```

Comment:
There are a _lot_ of typographic issues throughout this entire page that need to be fixed.

Example:

> the `|`, operator

…should be:

> the `|` (pipe) operator